src/animation/KeyframeTrackPrototype.js

```diff
@@ -336,12 +336,13 @@ KeyframeTrackPrototype = {
 
 		// flush last keyframe (compaction looks ahead)
 
-		times[ writeIndex ++ ] = times[ lastIndex ];
+		times[ writeIndex ] = times[ lastIndex ];
 
-		for ( var readOffset = lastIndex * stride, j = 0; j !== stride; ++ j )
+		for ( var readOffset = lastIndex * stride, writeOffset = writeIndex * stride, j = 0; j !== stride; ++ j )
 
 			values[ writeOffset + j ] = values[ readOffset + j ];
 
+		++ writeIndex;
 
```

Comment:
BTW is `this.times.length > 1` guaranteed?
If not, I think here should be

    if ( lastIndex > 0 ) {

        times[ writeIndex ] = times[ lastIndex ];
  
        for ( var readOffset = lastIndex * stride, writeOffset = writeIndex * stride, j = 0; j !== stride; ++ j )
  
            values[ writeOffset + j ] = values[ readOffset + j ];

        ++ writeIndex;

    }

correct?